rulesets/new-york-times-securedrop-ruleset.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="New York Times">
+	<target host="nytimes.securedrop.tor.onion" />
+	<rule from="^http[s]?://nytimes.securedrop.tor.onion"
+    to="http://ej3kv4ebuugcmuwxctx5ic7zxh73rnxt42soi3tdneu2c2em55thufqd.onion" />
```

Comment:
Actually, to `https://ej3kv4ebuugcmuwxctx5ic7zxh73rnxt42soi3tdneu2c2em55thufqd.onion`, which this redirects to.